### ISSUE TITLE
rereplicate should print a warning if abundance information is missing

### ISSUE THREAD

**Issue by frederic-mahe** [torognes/vsearch#339]
The option `--sizein` seems to be always implied when using the command `--rereplicate`:
```sh
## rereplicate: --sizein is always implied
DESCRIPTION="rereplicate: --sizein is always implied"
VSEARCH=$(which vsearch)
printf ">s;size=2;\nA\n" | \
    "${VSEARCH}" --rereplicate - --quiet --output -
printf ">s;size=2;\nA\n" | \
    "${VSEARCH}" --rereplicate - --quiet --sizein --output -
```
That makes sense. However, there should be a warning when abundance information is missing from one or all input sequences:
```sh
## rereplicate should print a warning if abundance information is missing
DESCRIPTION="rereplicate should print a warning if abundance information is missing"
VSEARCH=$(which vsearch)
printf ">s\nA\n" | \
    "${VSEARCH}" --rereplicate - --sizein --quiet --output /dev/null
```

**Comment by torognes** [comment]
Should be fixed by commit f1509898ce4f0f026c3211e256a81db3f5bfd827.

**Comment by torognes** [comment]
Fixed in version 2.12.0 just released.
